plugin/src/main/java/io/jenkins/plugins/coverage/metrics/steps/CoverageBuildAction.java

```diff
@@ -544,18 +544,25 @@ public String formatDelta(final Baseline baseline, final Metric metric) {
      * @param metric
      *         the metric to check
      *
-     * @return {@code true} if the trend is positive, {@code false} otherwise
+     * @return {@code roundedDeltaValue} if the trend is positive or negative, {@code 0} otherwise
      */
     @SuppressWarnings("unused") // Called by jelly view
-    public boolean isPositiveTrend(final Baseline baseline, final Metric metric) {
+    public double getTrend(final Baseline baseline, final Metric metric) {
         var delta = getDelta(baseline, metric);
         if (delta.isPresent()) {
-            if (delta.get().compareTo(Fraction.ZERO) > 0) {
-                return metric.getTendency() == MetricTendency.LARGER_IS_BETTER;
+            double deltaValue = delta.get().doubleValue();
+            // to apply rounding off for boundary delta values
+            double roundedDelta = Math.round(deltaValue * 100.0) / 100.0;
+            if (-0.1 < roundedDelta && roundedDelta < 0.1) {
```

Comment:
Sorry, my fault: percentages are stored in the interval [0,1], so it makes sense to compare 0.001. I also do not see the reason why you are rounding?